app/src/main/java/com/bc/gordiansigner/model/KeyInfo.kt

```diff
@@ -18,12 +19,17 @@ data class KeyInfo(
 
     @Expose
     @SerializedName("last_used")
-    var lastUsed: String,
+    var lastUsed: Date?,
 
     @Expose
     @SerializedName("is_saved")
     var isSaved: Boolean
 ): Parcelable {
+
+    companion object {
+        fun empty() = KeyInfo("", "", null, false)
```

Comment:
It should be `newEmptyInstance()` or `newDefaultInstance()`. `empty()` is like the for checking something.

Reply:
I think just `isEmpty()` is for checking. `Class.empty()` I saw should be considered to create an empty object, similar with `Collections.emptyList()` or `Collections.emptyMap()`, etc

Reply:
@hvthhien Collections is a generic stuff with a lot of derivation class like `List`, `Map`, ... so the `Collections.emptyList()` means create new instance of a list with empty elements, similarly for the rest. `KeyInfo` is in different manner, is really more specific. You can take a look on some kinds of Java naming convention (i think not only Java) when creating the new object instance as static method.